Tests/SentryTests/Integrations/SentryFramesTrackingIntegrationTests.swift

```diff
@@ -0,0 +1,49 @@
+import XCTest
+
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+class SentryFramesTrackingIntegrationTests: XCTestCase {
+
+    private class Fixture {
+        let options = Options()
+        let displayLink = TestDiplayLinkWrapper()
+        
+        init() {
+            options.dsn = TestConstants.dsnAsString(username: "SentryFramesTrackingIntegrationTests")
+        }
+    }
+    
+    private var fixture: Fixture!
+    
+    override func setUp() {
+        fixture = Fixture()
+    }
+    
+    func testFrameRenderingEnabled_MeasuresFrames() {
+        let sut = SentryFramesTrackingIntegration()
```

Comment:
M: it'd be nice if the fixture had a `getSut` method, so in case this class gets a new parameter or something, all the tests would no be changed, but rather the `getSut`